api/rds.proto

```diff
@@ -585,6 +588,18 @@ message RedirectAction {
   RedirectResponseCode response_code = 3 [(validate.rules).enum.defined_only = true];
 }
 
+message DirectResponseAction {
+  // Specifies the HTTP response status to be returned.
+  google.protobuf.UInt32Value status = 1 [(validate.rules).message.required = true];
```

Comment:
Can this be uint32 instead of google.protobuf.UInt32Value?

Reply:
Also, can we validate that it is >= 100 and < 600? (I don't know if that's even possible, but would be useful if it is)

Reply:
There's a precedent for using plain uint32 and range limits here:
https://github.com/envoyproxy/data-plane-api/blob/master/api/filter/http/fault.proto#L22

@htuch is that considered a good pattern to use? (I don't know the history behind why some declarations use the simple int types and others use the google.protobuf.* types.)

Reply:
This came up in the newly minted STYLE.md: https://github.com/envoyproxy/data-plane-api/blob/master/STYLE.md

"Use wrapped scalar types where there is a real need for the field to have a default value that does not match the proto3 default (0/false/""). This should not be done for fields where the proto3 defaults make sense. All things being equal, pick appropriate logic, e.g. enable vs. disable for a bool field, such that the proto3 defaults work, but only where this doesn't result in API gymnastics."

Reply:
Yep, +1 to @ggreenway comment above, we've been getting better at using unwrapped scalar types where it makes sense, so go for it.